docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+services:
+  website-niaefeup-frontend-dev:
+    build:
+      context: .
+      dockerfile: Dockerfile.dev
+    ports:
+      - 3000:3000
+    volumes:
+      - ./src:/usr/src/website-niaefeup-frontend/src
+      - ./static:/usr/src/website-niaefeup-frontend/static
+    environment:
+      - PORT=3000
+
+  website-niaefeup-frontend-storybook:
+    build:
+      context: .
+      dockerfile: Dockerfile.storybook
+    ports:
+      - 6006:6006
+    volumes:
+      - ./src:/usr/src/website-niaefeup-frontend/src
+      - ./static:/usr/src/website-niaefeup-frontend/static
+    environment:
+      - PORT=6006
+
+  website-niaefeup-frontend-test:
```

Comment:
This works well but I have to press ctrl+c after the tests execute. The test run is expecting me to press "q" to quit but the docker shell is not interactive. I think we could either start an interactive shell (`-it` flag with docker so there should be a way to do it with docker-compose) or check if there's a flag in vitest that exits automatically (less preferred because vitest detects changes and we might want the feature)